package.json

```diff
@@ -90,7 +90,7 @@
     "react": "16.13.1",
     "react-intl": "^2.8.0",
     "react-markdown": "4.3.1",
-    "react-native": "0.63.4",
+    "react-native": "0.64.2",
```

Comment:
@vsubhuman note that in general, it's not recommended to just bump the RN version without considering other modifications in the project that are included in the new target version.

The recommended way is to manually include all these changes (but maybe not all are critical or even necessary at all): https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.2

Reply:
Thank you, Vicente!